x-pack/plugins/infra/public/pages/metrics/inventory_view/hooks/use_snaphot.ts

```diff
@@ -44,7 +44,7 @@ export function useSnapshot(
     interval: '1m',
     to: currentTime,
     from: currentTime - 1200 * 1000,
-    lookbackSize: 20,
+    lookbackSize: 5,
```

Comment:
What does this change?

Reply:
It shortens the time range we are looking at for data, instead of looking at the last 20 minutes of data it will now look at the last 5 minutes of data. The caveat is that we use either use 60 seconds for the interval OR whatever `metricset.period` returns as long as it larger than 60 seconds. For `system.*` metricsets we use the last 60 seconds because they are shipped every 10 seconds by default. But for AWS EC2 metricsets we use the last 5 minutes because they are shipped every 300 seconds.

It doesn't make a huge improvement but at this point every little bit helps :D

Reply:
Ah ok so this means nodes that disappear will now stick around for an extra 5 "intervals" instead of an extra 20 "intervals", right?

Reply:
Yes